lib/CMakeLists.txt

```diff
@@ -15,7 +15,9 @@ ENDIF (HAVE_REST)
 
 # Place a default config file in a default location
 # CONFDIR normally expands to /usr/local/etc/
+# XXX: why is cogserver.conf installed if the default file name is opencog.conf?
```

Comment:
probably because the default should be changed to cogserver.conf.  The idea is to emphasize that this is a config file for the cogserver, and not for all of opencog in general.